docs/PLUGIN_CATALOG.md

```diff
@@ -137,6 +135,5 @@ There will always be more plugins we wish we had. To make sure others can contri
 | [#1196](https://github.com/intelsdi-x/snap/issues/1196) | publisher | Plugin Wanted: Publisher for Prometheus  |
```

Comment:
Makes sense. I think the duplication comes from our workflow. Usually the PR for plugins.yml includes the `Fixes #XX` for the wishlist item. Good catch 👍